app/annotationsList/annotationList.js

```diff
@@ -183,9 +183,19 @@ app.controller('AnnotationListCtrl', function($rootScope, $scope, $http, $routeP
                 goTermIds = goTermIds.concat(annotation.slimmedIds);
             }
 
-            var pos = annotation.geneProductId.indexOf(':');
-            if (pos !== -1) {
-                annotation.geneProductSimpleId = annotation.geneProductId.substring(pos + 1);
+            var firstSemicolonPosition = annotation.geneProductId.indexOf(':');
```

Comment:
Would split be useful here?

Reply:
this regexp might be useful if the extension_id (the third optional part after the second colon) contains colons:
```
/(.*?)\:(.*?):(.*)/
```